main.go

```diff
@@ -108,6 +109,26 @@ func main() {
 		return
 	}
 
+	scaledObjectMaxReconciles := 10
+	scaledObjectMaxReconcilesStr := os.Getenv("KEDA_SCALEDOBJECT_CTRL_MAX_RECONCILES")
+	if scaledObjectMaxReconcilesStr != "" {
+		scaledObjectMaxReconciles, err = strconv.Atoi(scaledObjectMaxReconcilesStr)
+		if err != nil {
+			setupLog.Error(err, "Invalid KEDA_SCALEDOBJECT_CTRL_MAX_RECONCILES")
+			return
```

Comment:
if you do want to bail out rather than falling back to a default (see the next comment), maybe `os.Exit(1)` here instead of `return`?

Reply:
you are right!